css/challenge_two.css

```diff
@@ -0,0 +1,50 @@
+#boxC {
+  width: 100px;
+  height: 100px;
+  background-color: Cyan;
```

Comment:
small things to note: always use hex colours (http://www.color-hex.com/) as browsers can sometimes interpret colour differently